From 2da2ddd3777ee5ab35b197786677eb3e0df27030 Mon Sep 17 00:00:00 2001
From: Simon Howard <fraggle@soulsphere.org>
Date: Sat, 15 Aug 2015 13:59:19 -0400
Subject: [PATCH 09/13] Remove duplicate variable definitions.

gametic and singletics are defined in d_loop.c, but there were still
some duplicate definitions of these variables left over from before
the game code was factored out.

This fixed #561 (thanks edward-san).
---
 src/heretic/d_main.c | 1 -
 src/heretic/g_game.c | 1 -
 src/hexen/g_game.c   | 1 -
 src/hexen/h2_main.c  | 1 -
 src/strife/g_game.c  | 1 -
 5 files changed, 5 deletions(-)

diff --git a/src/heretic/d_main.c b/src/heretic/d_main.c
index beb876c..6521aff 100644
--- a/src/heretic/d_main.c
+++ b/src/heretic/d_main.c
@@ -62,7 +62,6 @@ boolean respawnparm;            // checkparm of -respawn
 boolean debugmode;              // checkparm of -debug
 boolean ravpic;                 // checkparm of -ravpic
 boolean cdrom;                  // true if cd-rom mode active
-boolean singletics;             // debug flag to cancel adaptiveness
 boolean noartiskip;             // whether shift-enter skips an artifact
 
 skill_t startskill;
diff --git a/src/heretic/g_game.c b/src/heretic/g_game.c
index 1154925..6ad3860 100644
--- a/src/heretic/g_game.c
+++ b/src/heretic/g_game.c
@@ -102,7 +102,6 @@ player_t players[MAXPLAYERS];
 
 int consoleplayer;              // player taking events and displaying
 int displayplayer;              // view being displayed
-int gametic;
 int levelstarttic;              // gametic at level start
 int totalkills, totalitems, totalsecret;        // for intermission
 
diff --git a/src/hexen/g_game.c b/src/hexen/g_game.c
index e0a7afa..fd72d44 100644
--- a/src/hexen/g_game.c
+++ b/src/hexen/g_game.c
@@ -88,7 +88,6 @@ int RebornPosition;
 
 int consoleplayer;              // player taking events and displaying
 int displayplayer;              // view being displayed
-int gametic;
 int levelstarttic;              // gametic at level start
 
 char demoname[32];
diff --git a/src/hexen/h2_main.c b/src/hexen/h2_main.c
index 1e60c84..47c5e28 100644
--- a/src/hexen/h2_main.c
+++ b/src/hexen/h2_main.c
@@ -103,7 +103,6 @@ boolean debugmode;              // checkparm of -debug
 boolean ravpic;                 // checkparm of -ravpic
 boolean cdrom = false;          // true if cd-rom mode active
 boolean cmdfrag;                // true if a CMD_FRAG packet should be sent out
-boolean singletics;             // debug flag to cancel adaptiveness
 boolean artiskip;               // whether shift-enter skips an artifact
 int maxzone = 0x800000;         // Maximum allocated for zone heap (8meg default)
 skill_t startskill;
diff --git a/src/strife/g_game.c b/src/strife/g_game.c
index 906d3c9..c12e856 100644
--- a/src/strife/g_game.c
+++ b/src/strife/g_game.c
@@ -124,7 +124,6 @@ boolean         turbodetected[MAXPLAYERS];
  
 int             consoleplayer;          // player taking events and displaying 
 int             displayplayer;          // view being displayed 
-int             gametic; 
 int             levelstarttic;          // gametic at level start 
 int             totalkills, /*totalitems,*/ totalsecret;    // for intermission 
  
-- 
2.5.0

